.github/workflows/update-bootstrap-deploy-id.yml

```diff
@@ -0,0 +1,33 @@
+name: Update Bootstrap URL
+
+on:
+  workflow_dispatch:
+    inputs:
+      bootstrap_deploy_id:
+        description: 'Bootstrap Deploy ID'
+        required: true
+        type: string
+
+jobs:
+  update_bootstrap_url:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Replace Deploy ID
+        run:
+          find ./ -type f -exec sed -i 's/https:\/\/.*--edge-bootstrap\.netlify\.app/https://{{
+          $inputs.bootstrap_deploy_id }}--edge-bootstrap.netlify.app/g' {} \;
+
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v4
+        with:
+          title: 'chore: update edge-bundler-bootstrap'
+          body:
+            'Updates edge-bundler-bootstrap to [`{{ $inputs.bootstrap_deploy_id
+            }}`](https://app.netlify.com/sites/edge-bootstrap/deploys/{{ $inputs.bootstrap_deploy_id }}).'
+          commit-message: 'chore: update edge-bundler-bootstrap'
+          branch: update-edge-bundler-bootstrap
+          delete-branch: true
+          labels: 'type: chore'
+          team-reviewers: 'netlify/compute'
```

Comment:
Can we leave this up to the CODEOWNERS file?

Reply:
agreed, will be addressed in #45